code/game/objects/items/weapons/RCL.dm

```diff
@@ -16,7 +16,8 @@
 	w_type = RECYK_ELECTRONIC
 	melt_temperature = MELTPOINT_PLASTIC
 	origin_tech = "engineering=2;materials=4"
-	//var/active = 0 Depreciated; Leaving it because it is a useful framework tool if you want to make it automagically place on movement
+	var/max_amount = 90
+	var/active = 0
 	var/obj/structure/cable/last = null
 	var/obj/item/stack/cable_coil/loaded = null
 

```

Comment:
Does this not have a destroy proc? If you are going to keep references or objects inside of your object you NEED a destroy proc.

qdel the inner objects, then set them to null. For any references to objects outside null the references.